recipes/blaupause/meta.yaml

```diff
@@ -0,0 +1,78 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
```

Comment:
Can you strip out all the comments (that came from the original template)? They actually make this harder to work with.

Reply:
Thank you @dopplershift, the comments are now stripped out.